src/diem/testing/cli/click.py

```diff
@@ -41,6 +41,16 @@ def start_server(
 
 @click.command()
 @click.option("--target", "-t", default="http://localhost:8888", help="Target mini-wallet application URL.")
+@click.option("--stub-bind-host", "-h", default="localhost", help="The host the miniwallet stub server will bind to")
+@click.option(
+    "--stub-bind-port", "-p", default=None, help="The port the miniwallet stub server will bind to. Random if empty."
+)
+@click.option(
+    "--diem-account-base-url",
```

Comment:
--stub-diem-account-base-url

Reply:
looks like we should name this `--stub-access-url` or `--stub-server-url`, which means what is url a client can connect to, which will also be used as diem account base url.